src/components/KonnectorTile.jsx

```diff
@@ -70,33 +43,45 @@ export class KonnectorTile extends Component {
       t
     } = this.props
     const { domain, secure } = this.context
-    const hideKonnectorErrors = flag('hide_konnector_errors')
+
+    let status
+
+    if (konnector.available_version) status = STATUS.UPDATE
+    else if (isInMaintenance) status = STATUS.MAINTENANCE
+    else if (error || userError) status = STATUS.ERROR
+    else if (!accountsCount) status = STATUS.NO_ACCOUNT
+    else status = STATUS.OK
```

Comment:
should not be in a jsx component right? I know this is not part of the PR but KonnectorTile should not receive something like `isIsMaintenance` but just receive the konnector, and having an helper dealing with that to get the status ? 

Extracting this method can be a first step ?